src/test/java/org/mastodon/mamut/deeplineage/StartImageJ.java

```diff
@@ -38,8 +42,11 @@
  */
 public class StartImageJ
 {
+	private static final Logger logger = LoggerFactory.getLogger( MethodHandles.lookup().lookupClass() );
```

Comment:
I think something like: 
```private static final Logger logger  = LoggerFactory.getLogger(StartImageJ.class);```
would be more correct?

Reply:
There are pros and cons for both versions. While the version I am using, may be less readable to some developers, the version you suggest bears the risk of wrong copy & paste when transferring this code to a different class, cf.:

https://www.infoworld.com/article/2074637/portable-logger-names-with-java-7-method-handles.html
https://stackoverflow.com/questions/65117135/passing-methodhandles-lookup-lookupclass-vs-passing-class-to-getlogger-me

I am in favor of the version, I have suggested, since it can be universally used the same way in all classes with Java > 7.